ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/BlockHeaderBuilder.java

```diff
@@ -133,6 +133,33 @@ public static BlockHeaderBuilder fromHeader(final BlockHeader header) {
         .requestsHash(header.getRequestsHash().orElse(null));
   }
 
+  public static BlockHeaderBuilder fromHeader(
+      final org.hyperledger.besu.plugin.data.BlockHeader header) {
```

Comment:
BlockHeader implements org.hyperledger.besu.plugin.data.BlockHeader so I think you can change the param type for the existing method rather than duplicating it

line 118
  public static BlockHeaderBuilder fromHeader(final BlockHeader header) {